backend/api/v1/endpoints/search.py

```
@@ -4,6 +4,7 @@
 from typing import List
 from pydantic import BaseModel
 from models.post import Post
+from models.university import University
```

Ruff (0.8.2)

`models.university.University` imported but unused

Remove unused import: `models.university.University`

(F401)